lib/WP_Auth0_Lock10_Options.php

```diff
@@ -254,8 +254,7 @@ public function get_lock_options() {
       $extraOptions["auth"]["redirectUrl"] = $this->get_code_callback_url();
     }
 
-    $options_obj = array_merge_recursive( $extraOptions, $options_obj  );
-    $options_obj = array_merge_recursive( $options_obj , $extended_settings );
+    $options_obj = array_replace_recursive( $extraOptions, $options_obj, $extended_settings );
```

Comment:
So what is the logic here for duplicate entries. `extraOptions` are over ridden by `options_obj` but this can then be over ridden by `extended_settings`?

Reply:
@cocojoe - Correct.

Reply:
the naming seems odd. However, it is the same order as the original so fair enough